js/chat/chat.ts

```diff
@@ -369,6 +372,7 @@ class ChatContainer extends LightElement {
 
     if (message.chunk_type === "message_end") {
       lastMessage.removeAttribute("is_streaming");
+      lastMessage.setAttribute("content", message.content);
```

Comment:
Could you add context for this change? I'm fairly sure an [empty message gets send on `message_end`](https://github.com/posit-dev/py-shiny/blob/77a679/shiny/ui/_chat.py#L595), so I'm a bit surprised that this would change would do anything?